manifests/b/Burnaware/BurnAwareFree/14.4.0.0/Burnaware.BurnAwareFree.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Burnaware.BurnAwareFree
+PackageVersion: 14.4.0.0
+PackageLocale: en-US
+Publisher: Burnaware
+PackageName: BurnAware Free

```

```suggestion
PackageName: BurnAware Free 14.4 

```

To match the ARP table. Otherwise, LGTM!